sycl/doc/extensions/supported/sycl_ext_oneapi_bfloat16.asciidoc

```diff
@@ -104,12 +104,17 @@ namespace ext {
 namespace oneapi {
 
 class bfloat16 {
+using storage_t = uint16_t;
 
 public:
   bfloat16() = default;
   bfloat16(const bfloat16 &) = default;
   ~bfloat16() = default;
 
+  // Explicit conversion functions
+  static storage_t from_float(const float &a);
+  static float to_float(const storage_t &a);
```

Comment:
Why did these get added back?

Reply:
The headers use these when NVPTX is in effect. It looks like they are assumed to be available as public interfaces.